config.yml

```diff
@@ -29,9 +46,12 @@ steps:
         data:
           deploymentUrl: '%actions.pagesUrl.data.html_url%'
       - type: updateBranchProtection
-
   - title: Customize your homepage
     description: Open a pull request with changes to the homepage.
+    translations:
+      es:
+        title: Personaliza tu página inicial
+        description: Abre un pull request con cambios en la página inicial.
```

Comment:
🍺 Interesting. Is our idiom translating here as `Open` have the same meaning as `Abre`? I tend to think of these as `Starting` instead of `Opening` but I am not sure.